src/Files.Uwp/ServicesImplementation/FileExplorerService.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.Storage.Pickers;
+using Windows.System;
+using Files.Backend.Services;
+using Files.Sdk.Storage.LocatableStorage;
+using Files.Shared.Extensions;
+using Files.Uwp.Storage.WindowsStorage;
+
+#nullable enable
+
+namespace Files.Uwp.ServicesImplementation
+{
+    /// <inheritdoc cref="IFileExplorerService"/>
+    internal sealed class FileExplorerService : IFileExplorerService
+    {
+        /// <inheritdoc/>
+        public async Task OpenAppFolderAsync(CancellationToken cancellationToken = default)
+        {
+            await Launcher.LaunchFolderAsync(ApplicationData.Current.LocalFolder).AsTask(cancellationToken);
+        }
+
+        /// <inheritdoc/>
+        public async Task OpenInFileExplorerAsync(ILocatableFolder folder, CancellationToken cancellationToken = default)
+        {
+            await Launcher.LaunchFolderPathAsync(folder.Path).AsTask(cancellationToken);
+        }
+
+        /// <inheritdoc/>
```

Comment:
What does this stand for, is it a placeholder?

Reply:
It tells the VS that this method should inherit the comment from base method  (override method, interface method). I prefer it to stay as it also implies that the implemented method has a comment with it and if the code was changed, those changes should also be reflected in the comment